.github/workflows/build-backend.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Backend
+
+on:
+  push:
+    branches:
+      - 'dev'
+      - 'master'
+      - '40-create-a-dockerfile-for-frontend'
+  workflow_dispatch:
+jobs:
+  push_to_registry:
+    name: Build and push Backend Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Push to Docker Hub
```

Comment:
Out of curiosity... Erasme doesn't have its own registry ?

Reply:
in this situation it seemed more appropriate to publish the images on a public register, in order to allow the deployment by third parties